src/ol/controls/RoutingControl.js

```diff
@@ -65,6 +66,9 @@ class RoutingControl extends Control {
     /** @ignore */
     this.mot = options.mot || 'bus';
 
+    /** @ignore */
+    this.additionalRoutingParams = options.additionalRoutingParams || {};
```

Comment:
call it routingApiParams

Reply:
Alright. the prop is renamed

Reply:
What is the strategy? When do we expose parameters via JavaScript and when do we use the additional parameters?

Reply:
... forget it. The above parameters are just defaults. So only `mot` and `via` are exposed.
For me it's fine like this.